packages/shared/src/components/post/smartPrompts/SmartPrompt.tsx

```diff
@@ -68,7 +73,24 @@ export const SmartPrompt = ({ post }: { post: Post }): ReactElement => {
         </Tab>
 
         <Tab label={PromptDisplay.CustomPrompt}>
-          <input type="text" placeholder="Enter your custom prompt" />
+          <form
+            className="rounded-14 bg-surface-float"
+            onSubmit={onSubmitCustomPrompt}
+          >
+            <textarea
+              className="min-h-[9.5rem] w-full bg-transparent p-3"
+              placeholder="Enter your custom prompt"
```

Comment:
```suggestion
              placeholder="Write your custom instruction to tailor the post to your needs."
```

This is the copy from design